pytest_stress.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+import time
+
+SECONDS_IN_HOUR = 3700
```

Comment:
Is this a nit to be found? ;)

Reply:
Lol the name you mean?

Reply:
60 seconds * 60 minutes = 3600, not 3700 :)

Reply:
Oof! No wonder my hour long test was slightly longer 😂.

Good catch guys 👍.